client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -58,8 +62,20 @@ class SelectBusinessType extends Component {
 
 						<p>
 							{ translate(
-								'Google My Business lists your local business on Google Search and Google Maps. ' +
-									'It works for businesses that have a physical location or serve a local area.'
+								'{{a}}Google My Business{{/a}} lists your local business on Google Search and Google Maps. ' +
+									'It works for businesses that have a physical location or serve a local area.',
+								{
+									components: {
+										a: (
+											<a
+												href="https://www.google.com/business/"
+												target="_blank"
+												rel="noopener noreferrer"
+												onClick={ this.trackGoogleMyBusinessLinkClick }
+											/>
```

Comment:
It seems we are not consistent when it comes to decorating external links, but I think we should use the `<ExternalLink />` component here especially since we're using the same icon on the button.

As a reminder, here is an example on the [`Traffic` page](https://wordpress.com/settings/traffic):

<img width="551" alt="screenshot" src="https://user-images.githubusercontent.com/594356/37589683-4d566cec-2b66-11e8-8a4f-c645c6e8e738.png">